findme-location-riddles/src/ImageBucketRepository.py

```diff
@@ -27,20 +27,17 @@ def post_image_to_s3(self, image_base64, key):
         except ClientError as e:
             raise BadRequestError(f"Error saving image to bucket: {e}")
 
-    def get_image_from_s3(self, key):
+    def get_image_from_s3(self, key: str) -> dict:
```

Comment:
`encoded_image` below probably is a string
```suggestion
    def get_image_from_s3(self, key: str) -> str:
```